bigtable/docs/conf.py

```diff
@@ -306,12 +335,24 @@
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 # texinfo_no_detailmenu = False
 
+
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
     "python": ("http://python.readthedocs.org/en/latest/", None),
     "gax": ("https://gax-python.readthedocs.org/en/latest/", None),
+    "google-auth": ("https://google-auth.readthedocs.io/en/stable", None),
+    "google-gax": ("https://gax-python.readthedocs.io/en/latest/", None),
```

Comment:
Hmm, seems like we should be removing any reference to `gax` / `google-gax`, as we don't use it in the software since #4873.

Reply:
I just opened https://github.com/googleapis/gapic-generator/pull/2857 to remove the stale use of `google.gax.PageIterator` in generated docstrings.